alacritty/src/input.rs

```diff
@@ -492,26 +492,77 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
             };
 
             self.mouse_report(code, ElementState::Pressed);
-        } else if button == MouseButton::Left {
-            self.on_left_click();
         } else {
-            // Do nothing when using buttons other than LMB.
-            self.ctx.mouse_mut().click_state = ClickState::None;
+            // Reset click state if button has changed
```

Comment:
The comment is in accurate and missing a dot in the end. If anything it should mention that it updates the last button as well.

Reply:
I don't think the comment is inaccurate, it resets the click state so that the current click is only a single click. Updating the last button is just to be able to reset the click state again in the future, it shouldn't have any functional effect on anything else since we never read it except for this block.

The punctuation is incorrect though!